README.md

```diff
@@ -36,10 +36,11 @@ For clarity's sake all examples in this document use a customized bash prompt in
   - [Unstaged Edits](#unstaged-edits)
     - [I want to move my unstaged edits to a new branch](#i-want-to-move-my-unstaged-edits-to-a-new-branch)
     - [I want to move my unstaged edits to a different, existing branch](#i-want-to-move-my-unstaged-edits-to-a-different-existing-branch)
-    - [I want to discard my local, uncommitted changes](#i-want-to-discard-my-local-uncommitted-changes)
+    - [I want to discard my local uncommitted changes (staged and unstaged)](#i-want-to-discard-my-local-uncommitted-changes)
```

Comment:
Will this link still work?

Reply:
The link should work. We have a reference to that in the explanation section.